v2/spanner-custom-shard/src/main/java/com/custom/CustomTransformationWithShardForLiveIT.java

```diff
@@ -199,6 +199,21 @@ public MigrationTransformationResponse toSourceRow(MigrationTransformationReques
         throw new InvalidTransformationException(e);
       }
 
+      MigrationTransformationResponse response =
+          new MigrationTransformationResponse(responseRow, false);
+      return response;
+    } else if (request.getTableName().equals("Person")) {
```

Comment:
Can you add a todo and probably create a bug to rename this file? Looks like we are reusing this for Reverse replication Load tests, so the name should not contain `ForLive` and `IT` suffix.

Reply:
Added a todo, will take it up during my oncall rotation